app/components/bookings/overview/booking-overview.component.ts

```diff
@@ -1,9 +1,89 @@
+
 import {Component} from '@angular/core';
+import {NavController} from "ionic-angular/index";
+import {BookingComponent} from "../../../components/bookings/booking/booking.component";
+import {SessionService} from "../../../services/session/session.service";
+import {WorkpackageSelectionComponent} from "../workpackage-selection/booking-workpackage-selection.component";
+import {BookingService} from "../../../services/bookings/booking.service";
+import {Booking, BookingSession} from "../../../models/booking/booking";
+import {BookingSessionService} from "../../../services/bookings/booking-session.service";
+import moment = require("moment/moment");
+import Moment = moment.Moment;
 
 @Component({
-  templateUrl: 'build/components/bookings/overview/booking-overview.component.html'
+  templateUrl: 'build/components/bookings/overview/booking-overview.component.html',
+  directives: [BookingComponent],
+
 })
 export class BookingOverviewComponent {
-  constructor() {
+
+  private _bookings: Array<Booking>;
+  private _bokingSession: BookingSession;
+  private _pickedDate: Moment = moment(new Date());
+  private _pickedDateString: String = "";
+
+  private _hintMsg :String = "";
+
+  constructor( private _bookingService: BookingService,
+               private _navController: NavController,
+               private _bookingSessionService: BookingSessionService,
+               private _sessionService: SessionService
+   ) {
+      this._pickedDateString = this._pickedDate.format('DD.MM.YYYY');
+      this._loadBookings();
+      this._loadBookingSession();
+      this._sessionService.onProjectSelected.subscribe(() => this._loadBookings());
+  }
+
+
+  private _loadBookings(): void {
+
+    /*
+    TODO: first param is supposed to be moment, not null
+    * */
+    this._bookingService.get(/*moment(this.pickedDate.toString()*//*moment('2016-07-31')*/ null, null).subscribe((bookings: Array<Booking>) => {
+      this._bookings = bookings;
+    });
+  }
+
+
+  private _loadBookingSession(): void {
+
+    this._bookingSessionService.retrieve().subscribe((bookingSession: BookingSession) => {
+     if(bookingSession != null){
+       this._bokingSession = bookingSession;
+       this._hintMsg = "";
+     }
+     else if(this._bookings.length < 1) // no bookings an no live booking sessions
+      {
```

Comment:
opening and closing brackets should be on the same line with the `else` statement